llpc/builder/llpcBuilderBase.h

```diff
@@ -0,0 +1,62 @@
+/*
+ ***********************************************************************************************************************
+ *
+ *  Copyright (c) 2020 Advanced Micro Devices, Inc. All Rights Reserved.
+ *
+ *  Permission is hereby granted, free of charge, to any person obtaining a copy
+ *  of this software and associated documentation files (the "Software"), to deal
+ *  in the Software without restriction, including without limitation the rights
+ *  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ *  copies of the Software, and to permit persons to whom the Software is
+ *  furnished to do so, subject to the following conditions:
+ *
+ *  The above copyright notice and this permission notice shall be included in all
+ *  copies or substantial portions of the Software.
+ *
+ *  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ *  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ *  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ *  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ *  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ *  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ *  SOFTWARE.
+ *
+ **********************************************************************************************************************/
+/**
+ ***********************************************************************************************************************
+ * @file  llpcBuilderBase.h
+ * @brief LLPC header file: declaration of BuilderBase
+ ***********************************************************************************************************************
+ */
+#pragma once
+
+#include "llvm/IR/IRBuilder.h"
+
+namespace Llpc
+{
+
+using namespace llvm;
```

Comment:
I have started putting "using namespace llvm;" inside namespace Llpc { ... } in header files, to avoid the need for all the llvm:: prefixes. Do you think that is bad? Nicolai wasn't very keen on it either. :-) It's not as bad as putting "using namespace llvm;" at file scope in a header file.

Reply:
My issue is that this allows you to write stuff like `Llpc::Module`, `Llpc::StringRef`, etc., which gets very confusing quickly. You wouldn't write this code intentionally, but may miss it when you get suggestions from IDE for both code completion and find-definition.

Reply:
Headers are usually small, so I don't think prefixing everything with `llvm::` is a big burden.

Reply:
OK, I'll fix this instance here, and consider fixing the rest of them as part of the code style change.